### PR TITLE
feat: Adds in Standard Version

### LINKED ISSUE #28: Please add a CHANGELOG
I think we need a CHANGELOG: https://github.com/conventional-changelog/conventional-changelog

### PR DESCRIPTION
**Pull request by jhechtf**
Adds in npm script to add run standard-version.

In my personal projects I usually find a way to automate this step whenever something is pushed to the main branch. If I knew how to do that on Github, I would've done it, but I use Gitlab most often.

Closes #28

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -121,12 +122,13 @@
     "rollup-plugin-node-resolve": "^5.0.1",
     "rollup-plugin-svelte": "^5.0.3",
     "serialize-javascript": ">=3.1.0",
+    "standard-version": "^9.0.0",
     "svelte": "^3.18.2",
     "tarima": "^4.8.5",
-    "testcafe": "^1.9.4"
+    "testcafe": "^15.1.317922"
```
Comment (pateketrueke) [inline package.json:128]: I'm afraid of this change, please revert.

File: package.json [hunk 2]
```diff
@@ -121,12 +122,13 @@
...
   },
   "dependencies": {
     "abstract-nested-router": "^0.1.20",
     "query-string": "^6.8.3"
   }
-}
+}
```
Comment (pateketrueke) [inline package.json:134]: I think we should have an .editorconfig to enforce newlines at the end of the file. 🤔

**Review by pateketrueke — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jhechtf) [reply package.json:128]: Ok

File: package.json (hunk 2 above)
Comment (jhechtf) [reply package.json:134]: Sure, but I don't use editorconfig so I have no idea how to do that. I usually let ESLint do the styling.

**Review by pateketrueke — APPROVED** [review]
Blessed, thank you!

**Pull request merged**